jobmanager/WorkerThread.cpp

```diff
@@ -45,6 +53,30 @@ void WorkerThread::run()
     {
         logger::error("unknown exception", "WorkerThread");
     }
+
+    workerThreadHandler = nullptr;
+}
+
+bool WorkerThread::yield()
```

Comment:
put a comment here that it returns true if a job was processed. That will help when reading code at other places, without having excessive comments what is happening. The IDE will show me what the return value of yield means.

Reply:
Otherwise, brilliant.

Reply:
I added the comment on .h file